packages/hurumap-next/src/Map/utils.js

```diff
@@ -125,6 +126,7 @@ export const generateChoropleth = (colors, locations, mapType) => {
       code,
       count,
       fillColor: color,
+      color: choroplethProps.border_color,
```

Comment:
Should have a default, no?

Reply:
I have used the default color from the predefined styles, one thing I had missed is the color for inactive geometries i.e. those without any configured values, I have resorted into setting the default color coming in from the choropleth geometries.